local_pathfinding/local_path.py

```diff
@@ -25,6 +45,13 @@ def __init__(
 
 
 class LocalPath:
+    """The LocalPath sets and updates the local waypoints in between the
+
+    Attributes:
+        `_logger` (RcutilsLogger): The ROS logger of LocalPath.
+        `_ompl_path` (OMPLPath): ompl path object.
+        `waypoints` (List): Returns a list of coordinates (tuples (x, y)).
```

Comment:
Attributes don't return anything. Rather, `waypoints` is a list of coordinates that form the path.